lesson14/index.js

```diff
@@ -60,16 +61,29 @@ const showModal = () => {
   modal.classList.add("is-show");
 }
 
+const getInputNum = () => {
+  const inputNum = document.getElementById("js-input-number");
+  return new Promise((resolve)=> {
+    resolve(inputNum.value);
+  });
+}
+
 const init = async () => {
+  getInputNum().then((value) => {
+    console.log(value);
+  })
```

Comment:
IMO
get value in init function instead of getting inputNum.value in getInputNum function
and then pass it to init.

```js
fetchBtn.addEventListener("click", () => {
  const inputNum = document.getElementById("js-input-number");
  init(inputNum.value); 
});
```
Because getInputNum function is. function that get value from server side only.

Reply:
The reason why this is better is because it can be written more concisely?
I fixed it.Thank you.

2e7257c